src/main/java/com/ss/training/utopia/counter/entity/Booking.java

```diff
@@ -0,0 +1,128 @@
+package com.ss.training.utopia.counter.entity;
+
+import java.io.Serializable;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.IdClass;
+import javax.persistence.Table;
+
+/**
+ * @author Justin O'Brien
+ */
+@Entity
+@Table(name = "tbl_booking")
+@IdClass(BookingPk.class)
+public class Booking implements Serializable {
+
+	private static final long serialVersionUID = -3688288514632867852L;
+
+	@Id
+	@Column
+	private Long travelerId, flightId;
```

Comment:
Are these values longs in the database?

Reply:
They're integers, which use the same number of bits as Java integers, but are unsigned so their maximum value is twice as large (as I explained during our group meeting today).